src/pet.js

```diff
@@ -0,0 +1,125 @@
+// CONSTRUCTOR FOR PET OBJECTS //
```

Comment:
All the comments in this file are not necessary, the code is clear enough that they are not warranted. 
Comments are for explaining _how_ you are doing what you are and not _why_, or if something looks counter intuitive at first glance its a good idea to add one explaining why you are doing things in the way that you are. This is a far more valuable thing to provide to whoever is trying to understand and work on your code.